src/components/templates/Audience/Audience.scss

```diff
@@ -20,6 +20,7 @@ $spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
   height: 100%;
   padding: $spacing;
   min-width: fit-content;
+  width: 100%;
```

Comment:
Does this need to be explicitly added here? Will the `flex` in `src/components/organisms/WithNavigationBar/WithNavigationBar.scss`/similar not handle this already?

Reply:
Yes, we need, `.audience-container` is aligned width only for its content but not all parent container. Or use `flex: 1`

Reply:
Would `flex: 1` make it more versatile?

Reply:
updated to `flex: 1`